dashboards/TDinsight.md

```diff
@@ -349,6 +349,22 @@ Here's the metrics list:
 
 Currently only report login count per minute.
 
+## Upgrade
+
+Re install with `TDinsight.sh` will upgrade Grafana plugin and TDinsight dashboard.
+
+In other cases, users should install new TDengine data source plugin, then remove and install new TDinsight dashboard.
```

Comment:
In other cases, users should install the new TDengine data source plugin first, then remove and install the new TDinsight dashboard.

Is it better?

Reply:
lgtm.